### PR TITLE
Mh microfilms

### PR DESCRIPTION
**Pull request by mhallwac**
Functional test cases for Use Cases 1-3 on https://docs.google.com/spreadsheets/d/1kk7VT7o3iDd5yDaIgMOlVbg0WDpKQ69jv4_7Cyt4cgc/edit#gid=248186373

### CODE REVIEW

File: .byebug_history [hunk 1]
```diff
@@ -0,0 +1,2 @@
+q
+expect(page).to have_no_link 'A-Z Sort'
```
Comment (jeremyf) [inline .byebug_history]: The .byebug_history should be skipped.

Add the following to the project's `.gitignore` file:

```
.byebug_history
```

You can also set a global git ignore. Mine is in `~/.gitignore`

More details https://help.github.com/articles/ignoring-files/

File: spec/microfilms/functional/func_microfilms_spec.rb [hunk 2]
```diff
@@ -2,20 +2,55 @@
...
+      click_on('Ask a Librarian')
+    end
+  end
+
+  scenario 'Test 3: See List of Cities in A-Z Order' do
+    visit '/'
+    within('.facets') do
+      expect(page).to have_link 'City'
+      click_on('City')
+      expect(page).to have_link 'more »'
```
Comment (jeremyf) [inline spec/microfilms/functional/func_microfilms_spec.rb]: If we are clicking on this, do we need to say we expect it? I think you can remove this line.

File: spec/microfilms/functional/func_microfilms_spec.rb [hunk 3]
```diff
@@ -2,20 +2,55 @@
...
+    within ('#nav.sf-menu li.darker') do
+      expect(page).to have_link 'Ask a Librarian'
+      click_on('Ask a Librarian')
+    end
+  end
+
+  scenario 'Test 3: See List of Cities in A-Z Order' do
+    visit '/'
+    within('.facets') do
+      expect(page).to have_link 'City'
```
Comment (jeremyf) [inline spec/microfilms/functional/func_microfilms_spec.rb]: If we are clicking on city, do we need to say that it has the link

File: .byebug_history (hunk 1 above)
Comment (jeremyf) [inline .byebug_history]: @mhallwac To remove `.byebug_history` from being tracked do the following `git rm -f .byebug_history`

**Comment by h-parekh** [comment]
@jeremyf: ping

**Pull request merged**